__tests__/cmds/docs/index.test.ts

```diff
@@ -374,6 +374,63 @@ describe('rdme docs', () => {
     });
   });
 
+  describe('delete docs', () => {
+    it('should delete doc if file is missing and --deleteMissing option is used', async () => {
+      const versionMock = getAPIMock()
+        .get(`/api/v1/version/${version}`)
+        .basicAuth({ user: key })
+        .reply(200, { version });
+      const apiMocks = getAPIMockWithVersionHeader(version)
+        .get('/api/v1/categories?perPage=20&page=1')
+        .basicAuth({ user: key })
+        .reply(200, [{ slug: 'category1', type: 'guide' }], { 'x-total-count': '1' })
+        .get('/api/v1/categories/category1/docs')
+        .basicAuth({ user: key })
+        .reply(200, [{ slug: 'thisDocShouldBeMissingInFolder' }])
+        .delete('/api/v1/docs/thisDocShouldBeMissingInFolder')
+        .basicAuth({ user: key })
+        .reply(204, '');
+      const folderWithoutDocs = './__tests__/__fixtures__/ref-oas';
```

Comment:
Wait so is the idea that it'll delete every doc in ReadMe since the local folder is completely empty? Can you add a test case with a non-empty folder?

Reply:
Refactored both tests to use a folder with a document that should not be deleted